main.cc

```diff
@@ -170,6 +170,7 @@ struct convert<::object_storage_endpoint_param> {
             ep.config.aws->region = node["aws_region"].as<std::string>();
             ep.config.aws->key = node["aws_key"].as<std::string>();
             ep.config.aws->secret = node["aws_secret"].as<std::string>();
+            ep.config.aws->token = node["aws_token"].as<std::string>("");
```

Comment:
Nitpick (probably this patch isn't the place to fix it): I don't know why we picked names for these configuration options which aren't exactly the same as those in Amazon's configuration files. Those names are `aws_access_key_id`, `aws_secret_access_key` and `aws_session_token`. Yes, our names are shorter - but why?
I see below our environment variables do keep Amazon's naming - without trying to improve it.

Reply:
yeah, i concur with you. but let's create another PR for this change or discussion. was trying to be consistent here.

Reply:
#15529